tests/langchain/test_langchain_model_export.py

```diff
@@ -685,6 +666,7 @@ def test_langchain_agent_model_predict_stream():
     ]
 
 
+@pytest.mark.skipif(IS_LANGCHAIN_03, reason="Saving QAEvalChain does not work with LangChain 0.3.0")
```

Comment:
note: Unlike other legacy chains like `LLMChain` and `RetrieverQA`, the `QAEvalChain` is not officially deprecated. But in practice, this class is not getting much care lately (no update in 1 year aside from repo-wise refactoring: [commit history](https://github.com/langchain-ai/langchain/commits/c750600d3d0cd5fa9d2d1141b60ff17ee327b6db/libs/langchain/langchain/evaluation/qa/eval_chain.py)), so it should be fine to deprioritize.